source/_components/switch.knx.markdown

```diff
@@ -16,18 +16,22 @@ ha_iot_class: "Local Polling"
 
 The `knx` switch component is used as in interface to switching actuators.
 
+The `knx` component must be configured correctly, see [KNX Component](/components/knx).
+
 To use your KNX switch in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
 switch:
   - platform: knx
-    name: KNX Switch
-    address: 0/0/1
+    name: Kitchen.Coffee
+    address: '1/1/6'
 ```
+* **name** (*Optional*): A name for this devices used within Home Assistant.
+* **address**: KNX group address for switching switch on/off
```

Comment:
**Spelling & Grammar**:
- `switching switch on/off` -> `switching the switch on/off`